File: Backend/serviceokapi/app/Models/Establecimiento.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class Establecimiento extends Model
{
    
    /**
     * Indicates if the model should be timestamped.
     *
     * @var bool
     */
    public $timestamps = false;
    /**
     * The database table used by the model.
     *
     * @var string
     */
    protected $table = 'establecimiento';

    /**
    * The database primary key value.
    *
    * @var string
    */
    protected $primaryKey = 'EstablecimientoID';

    /**
     * Attributes that should be mass-assignable.
     *
     * @var array
     */
    protected $fillable = [
                  'ActividadID',
                  'CoordenadaX',
                  'CoordenadaY',
                  'Delivery',
                  'EntidadID',
                  'FlagActivo',
                  'Nombre',
                  'Permiso',
                  'RUC',
                  'Telefono',
                  'WhatsappURL'
              ];

    /**
     * The attributes that should be mutated to dates.
     *
     * @var array
     */
    protected $dates = [];
    
    /**
     * The attributes that should be cast to native types.
     *
     * @var array
     */
    protected $casts = [];
    
    /**
     * Get the Actividad for this model.
     *
     * @return App\Models\Actividad
     */
    public function Actividad()
    {
        return $this->belongsTo('App\Models\Actividad','ActividadID','ActividadID');
    }

    /**
     * Get the Representante for this model.
     *
     * @return App\Models\Representante
     */
    public function Representante()
    {
        return $this->belongsTo('App\Models\Representante','EntidadID','EntidadID');
    }

    /**
     * Get the calificacion for this model.
     *
     * @return App\Models\Calificacion
     */
    public function calificacion()
    {
        return $this->hasOne('App\Models\Calificacion','EstablecimientoID','EstablecimientoID');
    }



}

File: Backend/serviceokapiNetCore/WebApplication1/Models/Establecimiento.cs
using System;
using System.Collections.Generic;

namespace serviceOkapi.Models
{
    public partial class Establecimiento
    {
        public Establecimiento()
        {
            Calificacion = new HashSet<Calificacion>();
        }

        public int EstablecimientoId { get; set; }
        public string Nombre { get; set; }
        public int Telefono { get; set; }
        public string WhatsappUrl { get; set; }
        public int Delivery { get; set; }
        public int Permiso { get; set; }
        public string Ruc { get; set; }
        public string CoordenadaX { get; set; }
        public string CoordenadaY { get; set; }
        public int FlagActivo { get; set; }
        public int ActividadId { get; set; }
        public int RepresentanteId { get; set; }

        public virtual Actividad Actividad { get; set; }
        public virtual Representante Representante { get; set; }
        public virtual ICollection<Calificacion> Calificacion { get; set; }
    }
}
